<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceEditor</class>
 <widget class="QMainWindow" name="FaceEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Edit Shapes</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>711</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QCheckBox" name="autoActivateCheckBox">
         <property name="text">
          <string>Auto Edit Face Shape</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QListWidget" name="shapesListWidget">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QListWidget" name="controlsListWidget">
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="autoHideCheckBox">
         <property name="text">
          <string>Auto Hide other contrls and Focus</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="resetPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Reset Shape</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="updatePushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Update Shape</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="cancelPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel Edit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QListWidget" name="sdksListWidget">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="animCurveWidget" native="true"/>
       </item>
       <item row="5" column="1">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="editShapeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Edit Shape</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="mirrorShapeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Mirror Shape</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="newShapeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>New Shape</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Creat Shapes</string>
     </attribute>
     <widget class="QTextEdit" name="newExpressionTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>790</width>
        <height>30</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name the new Shape</string>
      </property>
     </widget>
     <widget class="QListWidget" name="newShapeCtrlsWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>790</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Controls in the Shape</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateCtrlsButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>790</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Controls Based on Selection</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createExressionButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>791</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFaceEditor">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpdate_Shape_List"/>
   </widget>
   <widget class="QMenu" name="menuFacial_Nodes_in_Scene">
    <property name="title">
     <string>Facial Nodes in Scene</string>
    </property>
    <addaction name="actionNONE_Update"/>
   </widget>
   <addaction name="menuFaceEditor"/>
   <addaction name="menuFacial_Nodes_in_Scene"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpdate_Shape_List">
   <property name="text">
    <string>Update Shape List</string>
   </property>
  </action>
  <action name="actionNONE_Update">
   <property name="text">
    <string>NONE (Update)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
